exchange/exchange-ps/exchange/Set-DistributionGroup.md

```diff
@@ -1131,7 +1131,10 @@ To add or remove owners without affecting other existing entries, use the follow
 
 Owners that you specify with this parameter are not added as group members. You need to manually add the owner as a member.
 
-**Note**: Group management in Outlook doesn't work when the owner is a mail-enabled security group. To manage the group in Outlook, the owner must be a mailbox or a mail user.
+> [!NOTE]
+> Group management in Outlook doesn't work when the owner is a mail-enabled security group. To manage the group in Outlook, the owner must be a mailbox or a mail user.
+>
+> If the _DL managed by_ or _owner_ is assigned to a security group, and when the owner login to `OWA options -> Distribution group`, they will not see the distribution list under "Distribution groups I own". If the _managed by_ or _owner_ is a normal user instead of a security group, they will be able to see them under "Distribution groups I own".
```

Comment:
```suggestion
> If the _DL managed by_ or _owner_ is assigned to a security group, and when the owner login to `OWA options -> Distribution group`, they will not see the distribution list under "Distribution groups I own". If the _managed by_ or _owner_ is a normal user instead of a security group, they will be able to see it under "Distribution groups I own".
```